.github/workflows/ci.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
       with:
         ghc-version: ${{ matrix.ghc-ver }}
         cabal-version: ${{ matrix.cabal }}
+    - name: System Dependencies
+      run: |
+        # Softfloat repo needs to be recursively cloned
+        cd submodules/softfloat-hs
+        git submodule update --init --recursive
```

Comment:
Can we replace the `cd` activities with just `git -C submodules/softfloat-hs submodule ...`?

Reply:
Oh yes, that's much better. I copied this part over from [`macaw`'s CI](https://github.com/GaloisInc/macaw/blob/1a2b9284f11eaa67420d29641afe3d32ae1418af/.github/workflows/ci.yaml#L57-L60), but I like your version more than what `macaw` is doing.